src/App.jsx

```diff
@@ -1,104 +1,22 @@
 import './App.scss';
+import './components/CommentList/CommentList.scss';
+import './components/UserInfo/UserInfo.scss';
+import './components/PostInfo/PostInfo.scss';
+import { PostList } from './components/PostList/PostList';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+const preparedPosts = postsFromServer.map(post => ({
+  ...post,
+  user: usersFromServer.find(user => user.id === post.userId),
```

Comment:
Would be good to create separate function for `usersFromServer.find(user => user.id === post.userId)`